arch/arm/core/cortex_m/tz/arm_core_tz.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arch/arm/cortex_m/cmsis.h>
+#include "cortex_m/tz.h"
+
+static inline void configure_nonsecure_vtor_offset(u32_t vtor_ns)
+{
+	SCB_NS->VTOR = vtor_ns;
+}
+
+static inline void configure_nonsecure_msp(u32_t msp_ns)
+{
+	__TZ_set_MSP_NS(msp_ns);
+}
+
+static inline void configure_nonsecure_psp(u32_t psp_ns)
+{
+	__TZ_set_PSP_NS(psp_ns);
+}
+
+static inline void configure_nonsecure_control(int spsel_ns,  int npriv_ns)
```

Comment:
inline can be dropped.

The compiler will inline it if appropriate.

Reply:
ok